src/ui/server/controllers/status.controller.ts

```diff
@@ -0,0 +1,22 @@
+import { Response } from 'express';
+import { DatabaseStatusController } from '../model/controller.model';
+import { DatabaseStatusDTO } from '../model/response.model';
+import { AbstractController } from './abstract.controller';
+import { controller, httpGet, response } from 'inversify-express-utils';
+import { ROUTE } from '../model/enums';
+
+enum STATUS_ROUTE {
+    ROOT = '/database'
+}
+@controller(ROUTE.VERSION + STATUS_ROUTE.ROOT)
+export class DefaultDatabaseStatusController extends AbstractController
```

Comment:
Strictly speaking it's just the Database controller - status is merely a subpath.
Please also change the filename.
It is likely that the overall name and/or path of this will change in the future, but it's good practice to name it according to the current state of our knowledge.